designsafe/apps/api/templates/designsafe/apps/api/agave/preview.html

```diff
@@ -95,8 +95,8 @@
         </p>
     {% endif %}
 
-    <script src="{% static 'vendor/jquery/dist/jquery.js' %}"></script>
-    <script src="{% static 'vendor/bootstrap-ds/js/bootstrap.js' %}"></script>
+    <script src="{% static 'vendor/jquery/dist/jquery.min.js' %}"></script>
```

Comment:
I think we can actually delete all of these preview templates. This is handled by Angular now.